src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,51 @@
+.calendar {
+  padding: 10px;
+
+  display: flex;
+  flex-flow: row wrap;
+
+  gap: $gap-between-days;
+  width: ($amount-of-day * $day-size) + (($amount-of-day - 1) * $gap-between-days) + 20px;
+
+  &__day {
+    position: relative;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    width: $day-size;
+    height: $day-size;
+    background-color: $day-color;
+
+    border: 1px solid $border-color;
+
+    transition: all 0.5s;
+
+    &:hover {
+      cursor: pointer;
+      background-color: $hover-day-color;
+      transform: translateY(-20px);
+    }
+
+    @for $i from 1 through 31 {
+      &:nth-child(#{$i})::before {
+        content: '#{$i}';
+
+        position: absolute;
```

Comment:
Do you need to set a position?
Even if you need this move to the selector itself, it's is the same for all of them so you don't need to do it inside loop

Reply:
thank you for reviewing!
I've just fixed mistakes.